hw/ip/kmac/data/kmac.hjson

```diff
@@ -4,6 +4,7 @@
 
 {
   name:               "kmac",
+  one_line_desc:      "Keccak / SHA-3 based keyed hash message authentication code engine with SCA and FI countermeasures",
```

Comment:
I think I wouldn't mention SHAKE because there is also cSHAKE. Technincally our KMAC supports SHA3-224, 256, 384, 512, SHAKE[128, 256] and cSHAKE[128, 256]. I think that would be too much info for the one liner. But your suggestion sounds good to me.

Reply:
Thanks both! After discussing with @vogelpi, we went for
> Accelerator for Keccak-based keyed hash message authentication code and SHA-3 hash functions; with SCA and FI countermeasures